x-pack/test/detection_engine_api_integration/utils/get_rule_so_by_id.ts

```diff
@@ -22,6 +23,6 @@ interface RuleSO {
  */
 export const getRuleSOById = async (es: Client, id: string): Promise<SearchResponse<RuleSO>> =>
   es.search({
-    index: '.kibana',
+    index: ALERTING_CASES_SAVED_OBJECT_INDEX,
```

Comment:
Nit: the JSDoc for this function still mentions the `.kibana` index. Could lead to confusion in the future

Reply:
The same is true for most modified utils in this folder

Reply:
Fixed in https://github.com/elastic/kibana/pull/154888/commits/1409810caa538b35b988931f13caa03f5100d2cc